packages/core/src/components/image-overlay/image-overlay.tsx

```diff
@@ -1,7 +1,49 @@
-import { Component, ComponentInterface, Element, forceUpdate, h, Host, Listen, State } from "@stencil/core";
+import {
+  Component,
+  ComponentInterface,
+  Element,
+  forceUpdate,
+  FunctionalComponent,
+  h,
+  Host,
+  Listen,
+  Prop,
+  State,
+} from "@stencil/core";
 import debounce from "debounce";
 import { createFocusTrap, FocusTrap } from "focus-trap";
 
+type ImageOverlayWijzigactie = "voegtoe" | "verwijder";
+
+const wijzigactieLabels: { [wijzigactie in ImageOverlayWijzigactie]: string } = {
+  verwijder: "Verwijderd",
+  voegtoe: "Toegevoegd",
+};
+
+const Dimmer: FunctionalComponent<{
+  active: boolean;
+  src: string | undefined;
+  alt: string | undefined;
+  ref: (element: HTMLHeadingElement | undefined) => void;
+  click: () => void;
+}> = ({ active, src, alt, ref, click }, children) =>
+  active &&
+  src &&
+  alt && (
+    <div class="dimmer">
+      <div class="wrapper" ref={ref}>
+        {children[2]}
```

Comment:
Er is niets op tegen om de items uit `children` onderdeel van de Functional Component props te maken. Dat maakt het duidelijker waar item 0, 1 en 2 voor zijn

Voor nu, voorwaarts!